changelog.txt

```diff
@@ -4,6 +4,7 @@
 * Fix - Paying with a saved card for a subscription with a free trial will now correctly save the chosen payment method to the order for future renewals.
 * Add - Payment Request Button support for US merchants (Apple Pay, Google Pay, Microsoft Pay, and the browser standard Payment Request API).
 * Update - Not passing level3 data for non-US merchants.
+* Fix - Deposit overview now displays placeholder information instead of keep loading when an error happen.
```

Comment:
Very minor nitpick: I think it would read better to say "Deposit overview now displays placeholder information instead of continuing to load when an error happens."

What do you think?

Reply:
Sounds better, thanks fro the advice! 🙂